src/main/resources/default/templates/biz/model/query-results.html.pasta

```diff
@@ -4,61 +4,97 @@
+
+                                                        <i:invoke
+                                                                template="/templates/biz/model/query-map-property.html.pasta"
+                                                                property="property.as(sirius.db.mixing.properties.BaseMapProperty.class)"
+                                                                entity="@entity"/>
 
-</w:page>
+                                                        <i:else>
+                                                            <i:if test="property.is(sirius.db.mixing.properties.BaseEntityRefListProperty.class)">
+                                                                <i:invoke
+                                                                        template="/templates/biz/model/query-ref-list-property.html.pasta"
+                                                                        property="property.as(sirius.db.mixing.properties.BaseEntityRefListProperty.class)"
+                                                                        entity="@entity"/>
 
-<script type="text/javascript">
-    $(document).ready(function () {
-        $('#editForm').keydown(function (e) {
-            if (e.keyCode === sirius.keys.KEY_ENTER) {
-                $('#editForm').submit();
-                return false;
-            }
-        });
-    });
-</script>
+                                                                <i:else>
+                                                                    <i:if test="property.is(sirius.db.mixing.properties.EnumProperty.class)">
+                                                                        <i:local name="enumValue"
+                                                                                 value="@property.getValue(entity)"/>
+
+                                                                        @toUserString(enumValue)
+                                                                        <span class="border-sirius-gray tycho-tag d-inline-block">
+                                                                    @enumValue.as(java.lang.Enum.class).name()
+                                                                </span>
+
+                                                                        <i:else>
+                                                                            @toUserString(property.getValue(entity))
+                                                                        </i:else>
+                                                                    </i:if>
+                                                                </i:else>
+                                                            </i:if>
+                                                        </i:else>
+                                                    </i:if>
```

Comment:
😱 :-P

Reply:
Didn't know a better way to do this in Tagliatelle as the switch block cant have that complex expressions, right?

So the only other way I see would be to introduce a method somewhere in the code that outputs the correct Query Tool property template for each property, which is then passed into a single invoke call. But this seems kinda overkill at the moment.